src/stormpath.login.js

```diff
@@ -14,7 +21,7 @@ angular.module('stormpath')
     $auth.authenticate($scope.formModel)
       .catch(function(response){
         $scope.posting = false;
-        $scope.error = response.data && response.data.error || 'XHR Error';
+        $scope.error = response.data && response.data.error || 'An error occured when communicating with server.';
```

Comment:
For a future PR, let's create an HTTP transform method which applies this default error message.